#request parameters

string joint_name
float32 angle
float32 speed
bool relative

--------------------------------
#response parameters

bool status